### ISSUE TITLE
The training code does not work well...

### ISSUE THREAD

**Issue by UdonDa** [NVlabs/FUNIT#40]
The result which is generated by the model is trained during 400 iterations.

I think it looks that the models fail to learn.
I modify `inplace=False in nn.ReLU and nn.LeakyReLU in blocks.py.` following this issue https://github.com/NVlabs/FUNIT/issues/23 

![gen_train_00000400_02](https://user-images.githubusercontent.com/25411643/79040609-97764980-7c24-11ea-87c7-76064098bf55.jpg)

Thanks.

**Comment by orpatashnik** [comment]
It takes a while for the model to generate some nice pictures. I trained with batch size of 16 pictures, and the pictures become reasonable after something like 20K iterations. It is possible to stabilize the training by reducing the learning rate, but then the final results look worse.

**Comment by UdonDa** [comment]
![gen_train_00050500_03](https://user-images.githubusercontent.com/25411643/79112785-59129300-7dba-11ea-8441-50027efa2325.jpg)

I can get the clear samples! thanks!
